src/content_delegate.js

```diff
@@ -220,6 +220,7 @@ ContentDelegate.prototype.handleDocketQueryUrl = function () {
         );
       } else {
         if (result.results) {
+          PACER.removeBanners()
```

Comment:
Curious...no semi-colon here?

Reply:
I created the commit before We fixed the issue with Prettier. After I merged the [main branch](https://github.com/freelawproject/recap-chrome/tree/main) with this one, the code formatter made a lot of changes to this file, so I created a [.git-blame-ignore-revs](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view) file to avoid messing with the blame view.

Thanks for mentioning that file during the meeting.